.claude/commands/create-eng-docs.md

```diff
@@ -215,19 +215,20 @@ Once confirmed, create the documentation file:
        > is good practice to add a **persistent**, **unique** id to the
        > component:
    - **Testing**:
+
 - Testing examples are auto-generated from `.docs.spec.tsx` files
-     - **Mandatory**: Include this text before the injection token:
-       > These examples demonstrate how to test your implementation when using
-       > [Component] in your application. The component's internal functionality
-       > is already tested by Nimbus - these patterns help you verify your
-       > integration and application-specific logic.
-     - Add injection token: `{{docs-tests: {component-name}.docs.spec.tsx}}`
-     - Create companion `.docs.spec.tsx` file with test sections (see Step 6.1)
-   - **Resources**:
-     - Link to Storybook (use "link-tbd" placeholder)
-     - **Internal component links must start with '/'** (e.g.,
-       `/components/inputs/searchinput`) to ensure absolute paths from root, not
-       relative to current page
+  - **Mandatory**: Include this text before the injection token:
+    > These examples demonstrate how to test your implementation when using
+    > [Component] within your application. As the component's internal
+    > functionality is already tested by Nimbus, these patterns help you verify
+    > your integration and application-specific logic.
```

Comment:
Besides a little wording tweakage, this is the only change made to this command. Everything else highlighted is a lie.